.vib/consul/goss/goss.yaml

```diff
@@ -0,0 +1,44 @@
+port:
+  tcp6:8400:
+    listening: true
+    ip:
+    - '::'
+file:   
+  /opt/bitnami/consul/conf/consul.json:
+    mode: "0644"
+    filetype: file
+    contains: 
+      - "8500"
+      - "8600"
+      - "8400"
+      - "8301"
```

Comment:
You may use environment variables (if defined) in favor of fixed values.

Also, these seem like ports for different protocols. If the configuration file defines them as something like `http: 8500` it may be interesting to check the protocols are assigned the right port:

```
{{ printf "/http.*%d/" .Env.CONSUL_HTTP_PORT_NUMBER }}
```